limited/apothecary

```diff
@@ -28,6 +28,7 @@ Choice `Cure` Chosen:
   • Remove `CureAvailable` from @Self
 Passive Start Day:
   • Change `Poisoned` value `1` to `Active` for @(AttrSelf:Poisoned)
+On Death: Remove `Poisoned:Self` from @(AttrSelf:Poisoned)
```

Comment:
why did we add this, this would make it so anyone poisoned would lose it when Apothecary dies when we are trying to implement the opposite